src/mono/sample/HelloWorld/Makefile

```diff
@@ -19,6 +19,7 @@ publish:
 	$(DOTNET) publish \
 	-c $(MONO_CONFIG) \
 	-r $(TARGET_OS)-$(MONO_ARCH) \
+	/p:SelfContained=true \
```

Comment:
I'd prefer setting this in the .csproj since it's not an optional property like the others.

Reply:
done.  also updated the console method body replacement sample